app/android.go

```diff
@@ -378,16 +378,41 @@ func driverShowFileOpenPicker(callback func(string, func()), filter *FileFilter)
 	}
 }
 
-func driverShowFileSavePicker(callback func(string, func())) {
+func driverShowFileSavePicker(callback func(string, func()), filter *FileFilter) {
 	fileCallback = callback
 
-	open := func(vm, jniEnv, ctx uintptr) error {
+	mimes := "*/*"
```

Comment:
Instead of duplicating this code can you please extract a method (i.e. `mimeStringFromFilter`)?